dali/operators/reader/loader/webdataset/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+collect_headers(DALI_INST_HDRS PARENT_SCOPE)
+collect_sources(DALI_WEBDATASET_SRCS)
+collect_test_sources(DALI_OPERATOR_TEST_SRCS PARENT_SCOPE)
+set(DALI_OPERATOR_SRCS ${DALI_OPERATOR_SRCS} ${DALI_WEBDATASET_SRCS} PARENT_SCOPE)
+set(DALI_OPERATOR_TEST_SRCS ${DALI_OPERATOR_TEST_SRCS} ${DALI_WEBDATASET_SRCS}
+    "${DALI_ROOT}/dali/util/file.cc"
+    "${DALI_ROOT}/dali/util/std_file.cc"
+    "${DALI_ROOT}/dali/util/mmaped_file.cc"
+    PARENT_SCOPE)
```

Comment:
This does not look right. Those files are already compiled in `libdali.so`, there's no need to compile them again. What is the problem you wanted to solve?